lib/screens/profile_screens/profile/interactor/viewmodels/profile_bloc.dart

```diff
@@ -36,13 +36,14 @@ class ProfileBloc extends Bloc<ProfileEvent, ProfileState> {
   Stream<ProfileState> mapEventToState(ProfileEvent event) async* {
     if (event is LoadProfileValues) {
       yield state.copyWith(pageState: PageState.loading);
-      final results = await GetProfileValuesUseCase().run();
-      yield ProfileValuesStateMapper().mapResultToState(state, results);
+      final Result<GetProfileValuesResponse> result = await GetProfileValuesUseCase().run();
```

Comment:
Do we need to specify those new long types here or I can keep using just `final results`?

Reply:
You dont need to specify the type.

Reply:
continue using final